packages/web-forms/src/components/RepeatInstance.vue

```diff
@@ -52,7 +52,7 @@ const menuItems: MenuItem[] = [
 
 </script>
 <template>
-	<FormPanel :title="label" :menu-items="menuItems" class="repeat" :label-number="instanceIndex + 1">
+	<FormPanel :title="label" :menu-items="menuItems" class="repeat-instance" :label-number="instanceIndex + 1">
```

Comment:
> With scoped, the parent component's styles will not leak into child components. However, a child component's root node will be affected by both the parent's scoped CSS and the child's scoped CSS. This is by design so that the parent can style the child root element for layout purposes.